src/modules/getMovies.js

```diff
@@ -0,0 +1,28 @@
+import commentClicked from './commentPopup.js';
+import display from './display.js';
+import { getLikes, postLikes } from './likes.js';
+
+const url = ('https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/eC8v1nWSCKC4m4T5bPlw/likes/');
```

Comment:
Hello @FrederickMih , thank you for the review.
We are having a hard time understanding the first point of this review:
" Based on the requirements of this project, confidential information about the API should be pushed to GitHub such as the key. You'll kindly fix this requirement."

Do you mean to say we should not push it to the GitHub? Because it is already pushed to the github.

Reply:
- Hi @yetemegn-telaye, I'm sorry for that!
- Confidential information such as the API key(`eC8v1nWSCKC4m4T5bPlw)` **shouldn't** be pushed to GitHub.


Great job! Your project has been approved :clap: 
Wish you all the best!